tests/Support/JsonApi/V1/Sites/SiteSchema.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace LaravelJsonApi\OpenApiSpec\Tests\Support\JsonApi\V1\Sites;
+
+use LaravelJsonApi\Core\Schema\Schema;
+use LaravelJsonApi\NonEloquent\Fields\Attribute;
+use LaravelJsonApi\NonEloquent\Fields\ID;
+use LaravelJsonApi\NonEloquent\Filters\Filter;
+use LaravelJsonApi\OpenApiSpec\Tests\Support\Entities\Site;
+
+class SiteSchema extends Schema
+{
+    public static string $model = Site::class;
+
+    public function fields(): iterable
+    {
+        return [
+            ID::make(),
```

Comment:
This field expects an integer as id by default, but you use a slug. It'll probably work, but is incorrect. Maybe just add `->matchAs('.+')` to accept everything?